### PR TITLE
Bugs Fixed

### PR DESCRIPTION
**Pull request by JoeyZhang00111**
fixed a few bugs in my Equals method.

### CODE REVIEW

File: main.java [hunk 1]
```diff
@@ -28,8 +28,10 @@ public static void main(String[] args) {
         System.out.println(emp3);
 
         System.out.println("\nDemonstrating equals method:");
-        System.out.println("Employee 1 equals Employee 2: " + emp1.equals(emp2));
-        System.out.println("Employee 2 equals Employee 3: " + emp2.equals(emp3));
+        System.out.println("Employee 1 equals Employee 2: "
+                + emp1.equals(emp2.getEmployeeNumber(), emp2.getHourlyWage(), (int) emp2.getHoursWorked()));
+        System.out.println("Employee 2 equals Employee 3: "
```
Comment (0x68656c6c6f20776f726c64) [inline main.java:33]: this line should return true

File: Employee.java [hunk 2]
```diff
@@ -19,7 +19,7 @@ public int getEmployeeNumber() {
     }
 
     public int getNextEmployeeNumber() {
-        return this.employeeNumber + 1;
+        return this.employeeNumber +1;
```
Comment (0x68656c6c6f20776f726c64) [inline Employee.java]: can be resolved by code clean up

File: Employee.java [hunk 3]
```diff
@@ -58,6 +58,18 @@ public double calculateGrossPay() {
     }
 
     // Constructors
+    public boolean equals(int employeeNumber, double hourlyWage, int hoursWorked) {
```
Comment (0x68656c6c6f20776f726c64) [inline Employee.java]: parameter can be just like `Employee employee`. Only one object need parsed. Can be refactored.

File: main.java (hunk 1 above)
Comment (JoeyZhang00111) [reply main.java:33]: number two and number three should return true now.

File: Employee.java (hunk 2 above)
Comment (JoeyZhang00111) [reply Employee.java]: ran code clean up

**Review by 0x68656c6c6f20776f726c64 — APPROVED** [review]

**Pull request merged**